source/content/guides/build-tools/01-introduction.md

```diff
@@ -21,104 +20,77 @@ image: buildToolsGuide-thumb
+- [<CustomIcon icon="bitbucket" /> BitBucket](https://bitbucket.org/product/) with [<CustomIcon icon="bitbucket pipelines" /> BitBucket Pipelines](https://bitbucket.org/product/features/pipelines)
 
-3. [Add an SSH key](/ssh-keys/) within your User Dashboard to enable passwordless access and avoid authentication prompts. Otherwise, provide your Pantheon Dashboard credentials when prompted.
+Build Tools projects also use [<CustomIcon icon="composer" /> Composer  <Popover title="CI Service" content="Composer is a PHP dependency manager that provides an alternative, more modern way to manage the external code used by a project." />](https://getcomposer.org) for PHP dependency management. If you haven't worked with Composer before, the [Pantheon Composer doc](/composer) is a great place to get started.
 
-4. [Generate a Machine Token](https://dashboard.pantheon.io/machine-token/create), then authenticate Terminus:
-
-      ```bash
-      terminus auth:login --machine-token=<machine-token>
-      ```
-
-5. Create the `$HOME/.terminus/plugins` directory if it does not already exist:
-
-      ```bash
-      mkdir -p $HOME/.terminus/plugins
-      ```
-
-6. Install the [Terminus Composer Plugin](https://github.com/pantheon-systems/terminus-composer-plugin):
+<Enablement title="Automation Training" link="https://pantheon.io/agencies/learn-pantheon?docs">
 
-    ```bash
-    composer create-project -n --no-dev -d $HOME/.terminus/plugins pantheon-systems/terminus-composer-plugin:~1
-    ```
+Master Composer, automated testing, and other advanced workflow concepts with help from our experts. Pantheon delivers custom workshops to help teams master our platform and improve their [WebOps](https://pantheon.io/webops) workflow.
 
-7. Install the [Terminus Drupal Console Plugin](https://github.com/pantheon-systems/terminus-drupal-console-plugin):
+</Enablement>
 
-    ```bash
-    composer create-project -n --no-dev -d $HOME/.terminus/plugins pantheon-systems/terminus-drupal-console-plugin:~1
-    ```
+## Build Tools Workflow
+Build Tools projects extend the [Pantheon workflow](https://pantheon.io/docs/pantheon-workflow) by adding an external Git provider and a Continuous Integration (CI) service.
 
-8. Install the [Terminus Build Tools Plugin](https://github.com/pantheon-systems/terminus-build-tools-plugin). Update the version number in this example from `2.0.0-beta12` to the current version:
+In this workflow, only files unique to the project are tracked as part of the external Git repository. The CI service then builds a production artifact, deploys the fully-built site to Pantheon, and runs a suite of automated tests.
 
-    ```bash
-    composer create-project --no-dev -d $HOME/.terminus/plugins pantheon-systems/terminus-build-tools-plugin:^2.0.0-beta12
-    ```
+This is quite different than working with the Pantheon-hosted Git repository for each site, where all code must be comitted.
 
-  <Alert title="Note" type="info">
+<BuildToolsStackSelectToolbar />
```

Comment:
Any components that start with `<BuildTools` will not work unless you also import them into this branch.

Reply:
@alexfornuto were the components that started with `<BuildTools` a part of the custom interactive build?

Reply:
![correctsir](https://user-images.githubusercontent.com/2349184/76780946-e93dc800-677b-11ea-9b34-dea949911f59.gif)

Reply:
Haha awesome, I'll remove them.